django/utils/log.py

```diff
@@ -189,3 +191,41 @@ def format(self, record):
 
         record.msg = msg
         return super(ServerFormatter, self).format(record)
+
+
+def log_response(message, *args, **kwargs):
+    """Log errors based on response status.
+
+    Log 5xx responses in error level and 4xx in warning level (unless level
+    is explicitly given as a keyword argument). The Response status_code and
+    the entire request are passed to logger extra parameter. This function
+    also avoid duplicated logs.
+
+    """
+
+    response = kwargs.get('response')
```

Comment:
I believe now they can be keyword-only arguments. In the first version of this PR I was using kwargs.pop to get the arguments I needed and then I was passing the remaining kwargs to the logging method.